lychee-bin/tests/cli.rs

```diff
@@ -364,10 +364,76 @@ mod cli {
             .assert()
             .success();
 
-        let expected = r#"{"total":11,"successful":11,"failures":0,"timeouts":0,"redirects":0,"excludes":0,"errors":0,"fail_map":{}}"#;
+        let expected = r#"{"total":10,"successful":10,"failures":0,"timeouts":0,"redirects":0,"excludes":0,"errors":0,"fail_map":{}}"#;
         let output = fs::read_to_string(&outfile)?;
         assert_eq!(output.split_whitespace().collect::<String>(), expected);
         fs::remove_file(outfile)?;
         Ok(())
     }
+
+    /// Test excludes
+    #[test]
+    fn test_exclude_wildcard() -> Result<()> {
```

Comment:
Thanks for adding these additional exclude tests and increasing the test coverage. ❤️